Dockerfile

```diff
@@ -84,14 +79,53 @@ ENV STABILITY ${STABILITY:-stable}
 ARG SYMFONY_VERSION=""
 
 # Download the Symfony skeleton and leverage Docker cache layers
-RUN composer create-project "symfony/skeleton ${SYMFONY_VERSION}" . --stability=$STABILITY --prefer-dist --no-dev --no-progress --no-scripts --no-plugins --no-interaction
+RUN composer create-project "symfony/skeleton ${SYMFONY_VERSION}" . --stability=$STABILITY --prefer-dist --no-dev --no-progress --no-scripts --no-plugins --no-interaction; \
```

Comment:
> I've tried with composer create-project symfony-skeleton:dev-master and it works.

Because there's `"minimum-stability": "dev"` in the unreleased versions of `symfony/skeleton`. But yeah, `minimum-stability` is dangerous and we should not encourage setting it to anything other than `stable` (the default). So we still should not use `--stability` here.